consumer/pdata/common.go

```diff
@@ -165,15 +165,14 @@ func (a AttributeValue) SetBoolVal(v bool) {
 	a.orig.BoolValue = v
 }
 
-// CopyFrom copies all the fields from the given AttributeValue.
+// CopyTo copies all the fields from the given AttributeValue.
 // Calling this function on zero-initialized AttributeValue will cause a panic.
-func (a AttributeValue) CopyFrom(av AttributeValue) {
-	akv := av.orig
-	a.orig.Type = akv.Type
-	a.orig.StringValue = akv.StringValue
-	a.orig.IntValue = akv.IntValue
-	a.orig.DoubleValue = akv.DoubleValue
-	a.orig.BoolValue = akv.BoolValue
+func (a AttributeValue) CopyTo(dest AttributeValue) {
+	dest.orig.Type = a.orig.Type
+	dest.orig.StringValue = a.orig.StringValue
+	dest.orig.IntValue = a.orig.IntValue
+	dest.orig.DoubleValue = a.orig.DoubleValue
+	dest.orig.BoolValue = a.orig.BoolValue
```

Comment:
I hope Go compiler is smart enough to only dereference `dest.orig` and `a.orig` once.

Reply:
It is in the cache anyway, and the benchmark did not change at all (1.5ns)